coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1855,12 +1855,22 @@ func InsertWorkspacePresetsAndParameters(ctx context.Context, logger slog.Logger
 
 func InsertWorkspacePresetAndParameters(ctx context.Context, db database.Store, templateVersionID uuid.UUID, protoPreset *sdkproto.Preset, t time.Time) error {
```

Comment:
I know we don't have too many tests covering `provisioner/terraform`, but this is `provisionerdserver`, should we cover changes introduced to `InsertWorkspacePresetAndParameters`?